oozie-to-airflow/integration/run_fs_test_with_airflow.bash

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
Can be removed with run-sys-test support.

Reply:
I updated a PR.